Composer/packages/electron-server/package.json

```diff
@@ -74,7 +74,7 @@
     "node-fetch": "2.6.1",
     "semver": "7.3.2",
     "tslib": "2.0.3",
-    "uuid": "^8.3.1"
+    "uuid": "7.0.0"
```

Comment:
This is the last version of `uuid` which supports deep imports. Deep imports are used in `@azure` packages so we can't change them.

I spent some time trying to update the packages and found that they still use deep imports in the latest versions.